assets/js/input.js

```diff
@@ -268,12 +268,21 @@
     }
 
     function getData() {
-      return $input.val();
+      var val = $input.val();
+
+      var unescaped = val.split(String.fromCharCode(92,92)).join(String.fromCharCode(92));
+      return unescaped;
     }
 
     function setData(data) {
       var str = JSON.stringify(data);
-      $input.val(str);
+      if(!str) {
+        $input.val('');
+        return;
+      }
+
+      var escaped = str.split(String.fromCharCode(92)).join(String.fromCharCode(92,92));
```

Comment:
The problem I can see with this is that by escaping "\" you could end up with "\\\\\\\" if you load, then save the value over again? Have you tested this

Reply:
Curious about benefits of `fromCharCode(92)` over a `replace(/\\/g,)`. Escaping is hell (alongside date conversions). Is this the holy grail? ✨ :